snuba/clickhouse/translators/snuba/mappers.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from snuba.clickhouse.translators.snuba import SnubaClickhouseStrictTranslator
+from snuba.clickhouse.translators.snuba.allowed import (
+    ColumnMapper,
+    SubscriptableReferenceMapper,
+)
+from snuba.query.dsl import array_element
+from snuba.query.expressions import Column, FunctionCall, SubscriptableReference
+
+
+@dataclass(frozen=True)
+class SimpleColumnMapper(ColumnMapper):
+    """
+    Maps a column with a name and a table into a column with a different name and table.
+
+    The alias is not transformed.
+    """
+
+    from_col_name: str
+    from_table_name: Optional[str]
+    to_col_name: str
+    to_table_name: Optional[str]
```

Comment:
indeed. I promised evan that I will let him commit #959 before changing columns for everyone since his PR has been there for longer. As soon as he merges I will adapt 963 if this will be already merged or vice versa.